system/modules/core/templates/elements/ce_accordion.html5

```diff
@@ -12,10 +12,10 @@
     <?php endif; ?>
 
     <?php if ($this->addImage): ?>
-      <figure class="image_container<?php echo $this->floatClass; ?>"<?php if ($this->margin): ?> style="<?php echo $this->margin; ?>"<?php endif; ?>>
+      <figure class="image_container<?php echo $this->floatClass; ?>"<?php if ($this->margin): ?> style="<?php echo $this->margin; ?>"<?php endif; ?> itemprop="associatedMedia" itemscope itemtype="ImageObject">
```

Comment:
You have specified the other `itemtype` attributes as `http://schema.org/<type>`. Why did you omit the `http://schema.org/` here?